package/crossplane.yaml

```diff
@@ -3,10 +3,10 @@ kind: Provider
 metadata:
   name: provider-digitalocean
   annotations:
-    source: github.com/khos2ow/provider-digitalocean
+    source: github.com/crossplane-contrib/provider-digitalocean
     license: Apache-2.0
     descriptionShort: |
       The DigitalOcean Crossplane provider enables infrastructure management for DigitalOcean.
 spec:
   controller:
-    image: khos2ow/provider-digitalocean-controller:VERSION
+    image: crossplane-contrib/provider-digitalocean-controller:VERSION
```

Comment:
Everything in this PR should be safe and transparent, except the official docker image for it, since it's currently missing in DockerHub.